internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -284,6 +285,66 @@ func (opts *Opts) askClusterOptions() error {
 	return nil
 }
 
+func (opts *Opts) defaultRegions() []string {
+	cloudProviders, err := opts.store.CloudProviderRegions(opts.ConfigProjectID(), opts.Provider, tier, false)
+
+	if err != nil {
+		return DefaultRegions[strings.ToUpper(opts.Provider)]
+	}
```

Comment:
Oh, I see you use it as a fallback, what are the chances that the api call will fail?

Reply:
Extremely low but I think that been prepared for the impossible is not a bad thing 😅

Reply:
low, but what would cause it to fail? would the user be able to continue with other API calls if this one fail?
My point here is, how likely is that if this fails then the next API calls will also fail and in turn we should probably not be so quiet about it

Reply:
yes, the user would be able to continue with other API calls with the current implementation.
I believe the user should be able to use the quickstart command even if the API call starts to fail. In addition, this strategy has been used also in other parts of the command.
Regarding what would cause this API to fail:
- changes in the atlas codebase which impacts the endpoint and have not been captured by the integration test
- no internet connection
- wrong API keys and/or projectID

For the last two points, the command will eventually fail

Reply:
> the command will eventually fail anyway.

Why wait?what's the point of asking so many questions if at the end you're going to fails?

Reply:
okay, I will remove this piece of code